src/implementations/twig/components/social-media-follow/social-media-follow.test.js

```diff
@@ -0,0 +1,75 @@
+import { merge, renderTwigFileAsNode } from '@ecl/test-utils';
+
+import demoData from '@ecl/specs-component-social-media-follow/demo/data';
+
+// Set fake paths for svgs to render for tests.
+demoData.links.forEach((link) => {
```

Comment:
We don't need this anymore, we did not have any path in the specs before, now i see you added it, so this will also improve the test by using a "realistic" path,` '/icon-social.svg'`